doc/faq.rst

```diff
@@ -378,6 +378,100 @@ password fields give no feedback, not even whether a character has been typed.
 Try using the number row instead, toggling Num Lock key, or SSHing into the
 virtual machine.
 
+SSH stops :code:`ch-image` build with interactive queries
+---------------------------------------------------------
+
+This often occurs during an SSH-based Git clone. For example::
+
+  $ cat ./Dockerfile
+  FROM alpine:latest
+  RUN apk add git openssh
+  RUN git clone git@github.com:hpc/charliecloud.git
+  $ ch-image build -t foo -f ./Dockerfile .
+  [...]
+  3 RUN ['/bin/sh', '-c', 'git clone git@github.com:hpc/charliecloud.git']
+  Cloning into 'charliecloud'...
+  The authenticity of host 'github.com (140.82.113.3)' can't be established.
+  RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
+  Are you sure you want to continue connecting (yes/no/[fingerprint])?
+
+At this point, the build stops while SSH waits for input.
+
+This happens even if you have :code:`github.com` in your
+:code:`~/.ssh/known_hosts`. This file is not available to the build because
+:code:`ch-image` runs :code:`ch-run` with :code:`--no-home`, so :code:`RUN`
+instructions can't see anything in your home directory.
+
+Solutions include:
```

Comment:
While it could seem like I'm stating the obvious here, it may not be obvious to all end-users...

I would recommend pointing out that most public repositories on most public Git hosting services can be cloned anonymously via HTTPS, so simply changing the URL that's being `git clone`'d might just be the quickest & easiest solution here.